app/widget-test/page.tsx

```
@@ -0,0 +1,40 @@
+'use client';
```

ESLint

[error] Replace `'use·client'` with `"use·client"`

(prettier/prettier)